VideoApi/VideoApi.Contract/Responses/ParticipantInHearingResponse.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace VideoApi.Contract.Responses
 {
-    public class JudgeInHearingResponse
+    public class ParticipantInHearingResponse
```

Comment:
Why the renaming? Is this applicable for all participants or just judges/staff members?

Reply:
I think it makes sense. Even if the usage is just for judges/staff member it's still a valid name.